test/seatalk.js

```diff
@@ -53,7 +53,7 @@ const should = chai.Should()
 chai.use(require('chai-things'))
 
 describe('seatalk', () => {
-  ;['$PSMDST,', '$STALK,'].forEach((prefix) => {
+  ;['$PSMDST', '$PSMDST_R', '$STALK,'].forEach((prefix) => {
```

Comment:
If I understood this PR correctly this test is not really relevant: it now tests the normal input format, but for `PSMDST_R`. Shouldn't we actually add a test where input does indeed have the extra `R` in the data?

Does the parser ever see the `C` data?

Reply:
I thought this test was testing all of the typical input formats. It previously had both $PSMDST and $STALK which are typical from various devices. The new Miniplex firmware adds the _R for all received data so this test just adds that option as well.

Also, the _C field may appear at the parser but I cannot imagine these would ever need to be parsed since they are sent messages meant to go onto the ST network.

Reply:
Several different things:
- You added support to a new data format, where the first (optional) data field has value`R`. That looks like it works, but none of the tests exercise it - you are running the tests that were already in place, with exact the same data fields as previously, no `R` field, just for an additional prefix
- the tests do not pass for `PSMDST` prefix because you deleted the trailing comma in the prefix value (see the diff above, it is highlighted)
- the tests do not pass for `PSMDST_R` prefix because there is no trailing comma in the new prefix value

Reply:
This should be corrected now.